<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetreGestion</class>
 <widget class="QMainWindow" name="fenetreGestion">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>873</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fenêtre de Gestion - MindPass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/MindPass-icone.png</normaloff>../ressources/MindPass-icone.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background: qradialgradient(spread:pad, cx:0, cy:1, radius:1.406, fx:0, fy:1, stop:0 rgba(244, 216, 148, 255), stop:1 rgba(255, 102, 102, 255));
}

#catTitle {
	color: white;
}

QLineEdit {
    background: rgba(255,255,255,100);
    border: none;
    border-radius: 5px;
    color: black;
    padding-left: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="corps_gestion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
    <item>
     <layout class="QVBoxLayout" name="Passwords" stretch="0,4,1">
      <item>
       <widget class="QLabel" name="titre_pwd">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mots de Passe</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_pwd">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>767</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_pwd">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>178</width>
           <height>765</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ajouter_pwd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="Sites">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    border: none;
    border-radius: 5px;
	background: rgba(255,255,255,50);
}

QLabel {
	background-color: transparent;
	background: transparent;
}
</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_sites">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>726</width>
         <height>834</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Categories" stretch="0,4,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="titre_cat">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Catégories</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_cat">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_cat">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>177</width>
           <height>767</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ajouter_cat">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuAide"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scrollArea_cat</tabstop>
  <tabstop>ajouter_cat</tabstop>
  <tabstop>Sites</tabstop>
  <tabstop>scrollArea_pwd</tabstop>
  <tabstop>ajouter_pwd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
